packages/fields-content/src/views/editor/Image.js

```diff
@@ -6,7 +6,11 @@ import { useStateWithEqualityCheck } from './hooks';
 
 const PopperRender = forwardRef(
   ({ update, alignment, isFocused, style, onAlignmentChange }, ref) => {
-    useLayoutEffect(update, [alignment]);
+    useLayoutEffect(() => {
+      // The update function is `null` on first render - we want to update this after the below component is mounted to have the correct handler
+      return update || undefined;
```

Comment:
This is making it run on cleanup which I don't think is intended?
```suggestion
      if (update) return update();
```